apps/transport/lib/jobs/RamboLauncher.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Transport.RamboLauncher do
+  @moduledoc """
+    A behavior for Rambo, with dynamic dispatching
+  """
```

Comment:
```suggestion
  @moduledoc """
  A behavior for Rambo, with dynamic dispatching
  """
```
Me semble qu'un linter n'aime pas sinon